src/View/State/ClearState.php

```diff
@@ -2,10 +2,13 @@
 
 namespace Statamic\View\State;
 
+use Statamic\Facades\URL;
+
 class ClearState
 {
     public function handle()
     {
         StateManager::resetState();
+        URL::clearExternalUrlCache();
```

Comment:
Feels weird for this to be in a "View" class.

An easy fix could be to just move this class from `src/View/State` to `src/Listeners` to make it more generic.